automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/lab/NotebookPanel.scala

```diff
@@ -0,0 +1,9 @@
+package org.broadinstitute.dsde.workbench.leonardo.lab
+
+
+trait NotebookPanel extends Toolbar with NotebookCell {
+
+  lazy val css = ".jp-Notebook.jp-NotebookPanel-notebook[id]"
```

Comment:
Is this being used anywhere?

Reply:
Not yet, but it'd definitely be useful in writing future tests so I thought I'd incorporate it.

Reply:
I would remove it for now, we can add it when we need it.